Extensions/FileSystem/filesystemtools.js

```diff
@@ -124,6 +124,87 @@ gdjs.fileSystem.makeDirectory = function (directory) {
   }
 }
 
+/**
+ * Save a string into a file.
+ * @param {string} text The string to be saved
+ * @param {string} savePath The absolute path on the filesystem
+ */
+gdjs.fileSystem.saveStringToFile = function (text, savePath) {
+  const fileSystem = typeof require !== 'undefined' ? require('fs') : null;
+
+  if (fileSystem) {
+    fileSystem.writeFile(savePath, text, (err) => {
+      if (err) {
+        console.error("Unable to save the text to path: '" + savePath + "': ", err);
+      }
+    });
+  }
+}
+
+/**
+ * Load a string from a file into a scene variable.
+ * @param {scenevar} scenevar The scene variable to store the string
+ * @param {string} loadPath The absolute path on the filesystem
+ */
+gdjs.fileSystem.loadStringFromFileSync = function (scenevar, loadPath) {
+  const fileSystem = typeof require !== 'undefined' ? require('fs') : null;
+
+  if (fileSystem) {
+    if (fileSystem.existsSync(loadPath)) {
+      try {
+        const data = fileSystem.readFileSync(loadPath, 'utf8');
+
+        if (data) {
+          scenevar.setString(data);
+        }
+      }
+      catch (err) {
+        console.error("Unable to load the file at path: '" + loadPath + "': ", err);
+      }
+    }
+  }
+}
+
+/**
+ * Load a string from a file into a scene variable asyncrounousely.
+ * @param {scenevar} scenevar The scene variable to store the string
+ * @param {string} loadPath The absolute path on the filesystem
+ */
+gdjs.fileSystem.loadStringFromFileAsync = function (scenevar, loadPath) {
+  const fileSystem = typeof require !== 'undefined' ? require('fs') : null;
+
+  if (fileSystem) {
+    if (fileSystem.existsSync(loadPath)) {
+      fileSystem.readFile(loadPath, 'utf8', (err, data) => {
+        if (err) {
+          console.error("Unable to load the file at path: '" + loadPath + "': ", err);
```

Comment:
Same, we could have an optional variable filled on error.
See also the definition of a error variable parameter for Facebook Instant games here: https://github.com/4ian/GDevelop/blob/5baf2d97352a003c1f995e254c8477ea94b95023/Extensions/FacebookInstantGames/JsExtension.js#L176

Note that even if it's optional, if the user does not specify a variable name, GDevelop will call the function with a default "empty variable". You can call `setString` on this variable, which will do nothing. (it's a special variable called "badVariable" which has all the methods redefined to do nothing).